### DISCUSSION TITLE
T CrB is about to go Nova. Can it be added to the free database?

### DISCUSSION THREAD

**Discussion by Jackalaq** [Stellarium/stellarium#3717] [Ideas]
Since this star is expected to go nova, then could it be included in the free version?

> From Wikipedia:
> **2016–present activity**
> 
> On 20 April 2016, the [Sky & Telescope](https://en.m.wikipedia.org/wiki/Sky_%26_Telescope) website reported a sustained brightening since February 2015 from magnitude 10.5 to about 9.2. A similar event was reported in 1938, followed by another outburst in 1946.[[22]](https://en.m.wikipedia.org/wiki/T_Coronae_Borealis#cite_note-sandt-22) By June 2018, the star had dimmed slightly but still remained at an unusually high level of activity. In March or April 2023, it dimmed to magnitude 12.3.[[23]](https://en.m.wikipedia.org/wiki/T_Coronae_Borealis#cite_note-23) A similar dimming occurred in the year before the 1946 outburst, indicating that it will likely erupt between April and September 2024.[[24]](https://en.m.wikipedia.org/wiki/T_Coronae_Borealis#cite_note-24)

**Comment by alex-w** [comment]
T CrB is included into Stellarium database a long time ago

**Comment by gzotti** [comment]
Free version as opposed to...? There is no Stellarium Desktop version to pay for.
